spec/system/vendor_system_spec.rb

```diff
@@ -69,13 +69,13 @@
       create(:purchase, :with_items, created_at: 1.week.ago, item_quantity: 15, amount_spent: 1, vendor: @vendor)
     end
 
-    scenario "Existing vendors show in the #index with some summary stats" do
+    it "can have existing vendors show in the #index with some summary stats" do
       visit url_prefix + "/vendors"
       expect(page).to have_xpath("//table/tbody/tr/td", text: @vendor.business_name)
       expect(page).to have_xpath("//table/tbody/tr/td", text: "25")
     end
 
-    scenario "Single vendor show semi-detailed stats about purchases" do
+    it "can have a single vendor show semi-detailed stats about purchases" do
       visit url_prefix + "/vendors/#{@vendor.to_param}"
       expect(page).to have_xpath("//table/tr", count: 3)
```

Comment:
This example fails (and probably the one after it)

My gut-check says that the layout for these was changed today by someone on the Theme Team, and that the table now uses a `tbody` tag. Go look at `/app/views/vendors/show.html.erb` and see what the `<table>` tag wraps the rows in now -- You might need to change this line and next line:
```suggestion
       expect(page).to have_xpath("//table/tbody/tr", count: 3)
```

Reply:
Fixed! Thank you for calling me out on that, I totally missed it!